internal/verify/test_fixture.go

```diff
@@ -142,19 +142,21 @@ func (tf *testFixture) runTestSuite(ctx context.Context, eng Checker, shouldRun
 		}
 
 		for _, action := range test.Input.Actions {
-			testKey := fmt.Sprintf("%s|%s|%s", test.Name.PrincipalKey, test.Name.ResourceKey, action)
+			testKey := fmt.Sprintf("%s|%s|%s|%s", test.Name.TestTableName, test.Name.PrincipalKey, test.Name.ResourceKey, action)
```

Comment:
We don't need to include the principal, resource and action here anymore. This check was added because of #1403. Because now the results are grouped by test name, that problem goes away. We only need to check that the test name is not duplicated.

Reply:
Given a test file like this;
```yaml
tests:
  - name: Test 1
    input:
      principals:
        - admin
        - user
      resources:
        - contact
      actions:
        - create

    expected:
      - principal: admin
        resource: contact
        actions:
          create: EFFECT_ALLOW
      - principal: user
        resource: contact
        actions:
          create: EFFECT_ALLOW
```
We have two tests (separated into two: one with principal `admin`, another one with `user`) having the same test name (`Test 1`). Since both have the `Test 1` name after separation, I didn't remove the logic.

Reply:
I see. In that case, the error message would look a bit strange because `testKey` already contains the test name in it. Maybe strip out the test name prefix from the `testKey` when producing the error message.